contracts/StorageAccessible.sol

```diff
@@ -63,11 +63,12 @@ contract StorageAccessible {
     function simulateDelegatecallInternal(
         address targetContract,
         bytes memory calldataPayload
-    ) public returns (bytes memory) {
+    ) public returns (bytes memory res) {
         (bool success, bytes memory response) = targetContract.delegatecall(
             calldataPayload
         );
-        revertWith(abi.encodePacked(response, success));
+        res = abi.encodePacked(response, success);
```

Comment:
I don't think this should change (is there a compiler error otherwise)? If we need to use `res` I'd again use it instead of response. But really the revert is populating the returned bytes so we shouldn't assign them ourselves.

Reply:
So I am not exactly sure what to do here otherwise... since I either have to name the return var... or 

<img width="865" alt="Screenshot 2020-12-02 at 11 18 59" src="https://user-images.githubusercontent.com/11778116/100860012-3e4b7680-3490-11eb-9216-8d04614e1320.png">

However, this method does not appear to have any non-reverting code paths...

Reply:
But I can do what you suggested as follows:

```js
res = response;
revertWith(abi.encodePacked(response, success));
```
Its just so weird to have to add this random, unnecessary line.